draft-looker-cfrg-bbs-signatures.md

```diff
@@ -1002,12 +1005,15 @@ Procedure:
 1.  let octets_to_hash be an empty octet string.
 2.  for el in input_array:
 3.      if el is an ASCII string: el = utf8(el)
-4.      if el is an octet string representing a public key: el_octs = el
-5.      else if el is an octet string:
-6.          if length(el) > 2^64 - 1, return INVALID
-7.          el_octs = I2OSP(length(el), 8) || el
-8.      else if el is a Point in G1: el_octs = point_to_octets_g1(el)
-9.      else if el is a Point in G2: el_octs = point_to_octets_g2(el)
+4.      if el is an octet string representing a public key:
```

Comment:
Is this used? Why not just pass it as a G2 point?

Reply:
We definitely could, but i like having the PK as an octet string. It's more intuitive for a developer IMO. Also updating this would be a bigger change (will need to update the `octets_to_pubkey` to something like `validate_pubkey` which I'm not sure how it would look like. Opened #219 to keep track of this.